ietf110/agenda.md

```diff
@@ -26,7 +26,7 @@
 * 5 min - Recap of interop at hackathon ([interop sheet](https://docs.google.com/spreadsheets/d/1D0tW89vOoaScs3IY9RGC0UesWGAwE6xyLk0l4JtvTVg/edit#gid=2079541679)) - *Lars Eggert*
 
 ### WG Items
-* 25 min - WGLC changes to [manageability](https://datatracker.ietf.org/doc/draft-ietf-quic-manageability/) and [applicability](https://datatracker.ietf.org/doc/draft-ietf-quic-applicability/). [Slides](https://datatracker.ietf.org/meeting/110/materials/slides-110-quic-quic-applicability-and-manageability-wglc-update-00) - *Mirja Kühlewind*
+* 25 min - WGLC changes to [manageability](https://datatracker.ietf.org/doc/draft-ietf-quic-manageability/) and [applicability](https://datatracker.ietf.org/doc/draft-ietf-quic-applicability/). [Slides](https://datatracker.ietf.org/meeting/110/materials/slides-110-quic-quic-applicability-and-manageability-wglc-update-00) - *Brian Trammell*
```

Comment:
Every word containing `(.*)man(.*)` is not inclusive? Interesting...

Reply:
Well, those regexes are a work in progress. Hopefully fixed now.

Reply:
https://github.com/jpoehnelt/in-solidarity-bot/issues/217